### PR TITLE
Move default.py into its own file

### LINKED ISSUE #311: Move the default config into a file of its own
Why do we even have this inline?

### PR DESCRIPTION
**Pull request by Androbin**
Fixes #311

### CODE REVIEW

**Comment by Androbin** [comment]
@sushain97 Friendly ping.

File: phenny [hunk 1]
```diff
@@ -28,80 +28,11 @@ def check_python_version():
...
-    # default = 300 (5 hrs))
-    pester_after_dismiss = 300
-
-    sf_issues_url = ''
-
-    # EOF
-    """), file=f)
-    f.close()
+def create_default_config(fn):
+    os.system('cp {:} {:}'.format(
```
Comment (shardulc) [inline phenny]: `cp` might not always be available on the system.

Comment (Androbin) [reply phenny]: Good point. Let's go for a more verbose version.

**Comment by Androbin** [comment]
Build looks good.

**Comment by shardulc** [comment]
Looks good to me. My only concern is that there are two `default.py`s: one in the bot's directory, and one in `~/.phenny`. No matter what message we give users, it is confusing which file to edit if you want to see changes in your bot. I think calling it `default_template.py` would do; do you have any ideas?

**Comment by Androbin** [comment]
I would rather call it `default.example.py` or `default.py.example`.
Just like [apertium-html-tools/config.conf.example](https://github.com/goavki/apertium-html-tools/blob/master/config.conf.example)

**Comment by Androbin** [comment]
Which variant would you prefer?

**Comment by shardulc** [comment]
`default.py.example` seems better.

**Comment by Androbin** [comment]
Anything else to consider?

**Comment by shardulc** [comment]
Looks good to me. However I feel like we should wait for @jonorthwash on this.

**Comment by Androbin** [comment]
@jonorthwash Friendly ping.

**Comment by jonorthwash** [comment]
Agreed!

**Pull request merged**